config/jobs/kubernetes-sigs/azuredisk-csi-driver/azuredisk-csi-driver-periodics-config.yaml

```diff
@@ -0,0 +1,890 @@
+periodics:
+  kubernetes-sigs/azuredisk-csi-driver:
+  - name: periodic-24h-azuredisk-csi-driver-verify
+    decorate: true
+    decoration_config:
+      timeout: 4h
+    interval: 24h
+    path_alias: sigs.k8s.io/azuredisk-csi-driver
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210312-67f589a-master
+        command:
+        - runner.sh
+        args:
+        - make
+        - verify
+        securityContext:
+          privileged: true
+    annotations:
+      testgrid-dashboards: provider-azure-azuredisk-csi-driver, provider-azure-periodic
+      testgrid-tab-name: periodic-24h-azuredisk-csi-driver-verify
+      description: "Periodic (24h): Run code verification tests for Azure Disk CSI driver."
+      testgrid-num-columns-recent: '30'
+  - name: periodic-24h-azuredisk-csi-driver-e2e-single-az
```

Comment:
I would suggest just run `periodic-24h-azuredisk-csi-driver-e2e-single-az` one test in the beginning to make sure periodic test run is stable enough, we don't need to run every e2e tests, and also they are not stable enough now.

Reply:
Fine running one set for now, but we should plan on enabling everything once we have more data.